lib/chef/mixin/shell_out.rb

```diff
@@ -101,7 +101,7 @@ def shell_out_with_systems_locale!(*args, **options)
       end
 
       def a_to_s(*args)
-        Chef.deprecated(:shell_out, "a_to_s is deprecated use shell_out with splat-args")
+        # Chef.deprecated(:shell_out, "a_to_s is deprecated use shell_out with splat-args")
```

Comment:
Why are we commenting out this deprecation warning? If we don't need it we should remove it entirely since we can always restore it from version control.

Reply:
Also not sure we can make available what what previously made known as deprecated. We should instead use the splat-args as mentioned in the suggestion for our changes.